CommunicationLayer/CommDeviceControl/RadioCommDevice.cpp

```diff
@@ -34,7 +34,7 @@ bool RadioCommDevice::connectToDataSource()
     }
     else
     {
-        qDebug() << serialPort_.errorString();
+        qInfo() << serialPort_.errorString();
```

Comment:
I wouldn't say so for initialization. Because of how this is written, it's a supported use case to not have a connection on startup. Usually if you have an error or a critical message, that means the normal flow of logic does not continue. This is not the  case in this block.

Reply:
I will change it to warning